src/tiled/objectselectionitem.cpp

```diff
@@ -127,14 +128,17 @@ static Preferences::ObjectLabelVisiblity objectLabelVisibility()
 }
 
 
-class MapObjectOutline : public QGraphicsItem
+// MapObjectOuline inherits from QObject to use timer signals
+class MapObjectOutline : public QObject , public QGraphicsItem
 {
 public:
     MapObjectOutline(MapObject *object, QGraphicsItem *parent = nullptr)
         : QGraphicsItem(parent)
         , mObject(object)
     {
         setZValue(1); // makes sure outlines are above labels
+        connect(updateTimer, &QTimer::timeout, this, [this] {this->update();});
+        updateTimer->start(100);
```

Comment:
I think 250ms works better.

Actually using marching ants is more common in graphics applications like GIMP and Photoshop, where the selection area is important to be aware of. In Inkscape there is no animation, for example. However, I think as long as it is at a slow pace, it's a nice thing to add.